complete/src/main/java/com/example/restservice/controller/model/token/TokenDetailResponse.java

```diff
@@ -11,6 +11,7 @@
 public class TokenDetailResponse {
 
   String tokenId;
+  String tokenNumber;
```

Comment:
is there a good reason to use `String`? Otherwise lets use Integer?

Reply:
Nothing specific. Converting to int